### PR TITLE
Add 2021 Aerials Screensavers and fix reported bugs

### LINKED ISSUE #48: [v3.0.1] Settings > Offline mode: 'Download Videos' in alphabetical order, but 'All' option isn't listed first
Just a little nitpick with this section-- 'Africa Night' and 'Africa and the Middle East' options appear before 'All' in the listing.

Expecting to see 'All' listed as first option.

### LINKED ISSUE #49: Offline download of some targets fail when HEVC is not enabled
Attempting to "Download Videos" for the following targets results in an error and failure to download:
* All
* California to Vegas
* Hong Kong

On the "Aerial Screensavers" tab I have:
```
Show start notification [ X ]
Show preview windows [ ]
Enable 4K (default 1080p) [ ]
Enable HDR [ ]
Enable HEVC (required for 4K/HDR) [ ]
```

Kodi.log when attempting to download from the "All" target is below.  Similar output when using the other two targets.
```
2021-04-29 06:11:40.006 T:153130   ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'TypeError'>
                                                   Error Contents: argument of type 'NoneType' is not iterable
                                                   Traceback (most recent call last):
                                                     File "/usr/share/kodi/addons/screensaver.atv4/entrypointscript.py", line 13, in <module>
                                                       atv.run(len(sys.argv) > 1)
                                                     File "/usr/share/kodi/addons/screensaver.atv4/resources/lib/atv.py", line 163, in run
                                                       offline()
                                                     File "/usr/share/kodi/addons/screensaver.atv4/resources/lib/offline.py", line 64, in offline
                                                       if "https" in url:
                                                   TypeError: argument of type 'NoneType' is not iterable
                                                   -->End of Python script error report<--
                                                   
2021-04-29 06:11:40.028 T:153130    INFO <general>: Python interpreter stopped
```

### PR DESCRIPTION
**Pull request by kubedzero**
Hope all is well and happy new year 🎉  

- I checked in on the state of tvOS and Aerials and found that there were a bunch of new scenes.
  - https://ko-fi.com/post/9-new-videos-available-C0C07GFG2 notes 6 Iceland and 3 Scotland from December 13, 2021
  - https://appleinsider.com/articles/21/09/21/apple-tv-gets-16-new-nature-themed-aerial-screen-savers-with-tvos-15 notes 4 Patagonia, 7 Yosemite, and 5 Grand Canyon videos from September 21, 2021
- I checked the https://sylvan.apple.com/Aerials/resources.tar file but didn't see any updates, meaning it's identical to the file currently committed from early 2021 in this codebase (and customized with deprecated H.264 and other videos)
- I did some looking around and figured out that https://sylvan.apple.com/Aerials/resources-15.tar has updated JSON once unpacked, over 1000 lines more!!! I guess they're appending the tvOS version number now. 
- Differences in the new tarball
	- At the top level, the old file had an `assets` array, the `initialAssetCount` key, and the `version` key. The new version has a `categories` array
	- Inside each of the items in the original assets array (where we source the video URLs) had `pointsOfInterest`, `accessibilityLabel`, `id`, `clipNumber` `url-1080-HDR`, `url-1080-SDR`, `url-4K-SDR`, `url-4K-HDR`, `url-1080-H264`. Now there is a `categories` array and a `shotID`, and the `clipNumber` is no longer used. They seem to differentiate based on shotID and ID alone now, with the accessibilityLabel being the same between them

I took these changes and merged them into the local `entries.json` file, and also updated the URL to point at the latest version of the Resources tarball. Along with this, I updated the toggle settings to cover missing locations (and the new locations) and added a big block of translations for all three languages as well. 
____

I also checked the issues folder:
- Fixes https://github.com/enen92/screensaver.atv4/issues/49 which was due to unsafe handling of a None return type when looking up the best-fit URL. Now it should play properly
- Fixes https://github.com/enen92/screensaver.atv4/issues/48 which was due to sorting All with the rest of the locations. All is now prepended and should always be listed first

## Testing

- Ran the `entrychecksumgenerator.py 2` to confirm the resources.tar file can still be downloaded from Apple, and that the entries.json can be retrieved from inside it
- Ran the `entrychecksumgenerator.py 1` to test downloading all videos. I validated that the MD5s matched from the original `checksums.json` from 2021, so any pre-existing URLs have been validated to retrieve the exact same content as before.
  - I also validated that this script downloaded the full set of 116 videos each for 4K SDR, 4K HDR, 1080p SDR, 1080p HDR, and 122 videos for 10800 SDR H.264 for 586 videos total
- For testing within Kodi, I tested on Kodi 19.3 Matrix on MacOS 12.0.1 Monterey, as well as Windows 10 1907
- Feel free to test locally for a while before merging, I'll be doing the same and will update this PR if I find any issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/e5d8dc168cf940a385d1a47837fe7596)](https://www.codacy.com/app/92enen/screensaver.atv4?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=enen92/screensaver.atv4&amp;utm_campaign=Badge_Grade)
```
Comment (kubedzero) [inline README.md:1]: This link was no longer working, I'm not familiar with Codacy but it gave me a 404 when I tried to go here.

**Comment by graysky2** [comment]
@kubedzero - Trying your PR now, seems to be working (downloading 28 new vids).  Interestingly, I could not apply this PR directly to v3.0.2 because that tarball lacks `entrychecksumgenerator.py`

I think it's due to https://github.com/enen92/screensaver.atv4/blob/master/.gitattributes#L9

@enen92 - Do you want to remove that line?

**Review by enen92 — APPROVED** [review]

**Comment by enen92** [comment]
@graysky2 that line is important since the file is not to be pushed to the repo when archiving the add-on.

I'll tag a new release shortly, thanks very much @kubedzero

**Comment by ValZapod** [comment]
BTW, those new videos here https://gist.github.com/h0wardch3ng/0f08dbcc3bc059186a3af98889e7c218 are actually using Dolby Vision IPTPQc2 colorspace, so it is actually real profile 5, while before it was just tagged as profile 5 because Apple TV does not support actual profile 8 which that was. mpv can play those videos nicely :_)

**Pull request merged**